.yamllint.strict

```diff
@@ -0,0 +1,57 @@
+extends: default
+
+rules:
```

Comment:
why are we enforcing indentation rules? i'm fine with the file passing validation regardless of how many spaces are used to indent

Reply:
will await feedback from this [comment](https://github.com/dreadnode/robopages/pull/7#discussion_r1824723559) since you may want to remove it altogether. 
that's fair and the reason was that i was mirroring [exec.yml](https://github.com/dreadnode/robopages/blob/1aab8d03f96a34fcf1a33511484c0d04083f81d3/coding/python/exec.yml) as an example accepted format to match against